File: flutter-main/flutter-main/Inheritence/p25.dart
class Demo{
	final int? x;
	final String? str;
	const Demo(this.x,this.str);
				
}
void main(){
	Demo obj1 = new Demo(10,"Sachin");
	Demo obj2 = new Demo(18,"Virat");
	
	print(obj1.hashCode);
	print(obj2.hashCode);
}

File: java-main/java-main/NestedforloopAssi/p25.java
/*
A B C D
B C D
C D
D
*/

class Demo{
    public static void main(String[] args) {
        char[] letters = {'A', 'B', 'C', 'D'};
        int n = letters.length;

        for (int i = 0; i < n; i++) {
           
            for (int j = i; j < n; j++) {
             
                System.out.print(letters[j] + " ");
            }
          
            System.out.println();
        }
    }
}
